<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartIntervalSelectionWidget</class>
 <widget class="QWidget" name="ChartIntervalSelectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>32</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Интервал с</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateTimeEdit" name="dateTimeEdit_intervalBegin">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Время начала выделенного интервала</string>
     </property>
     <property name="displayFormat">
      <string>dd.MM.yyyy H:mm:ss</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="timeSpec">
      <enum>Qt::UTC</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>по</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateTimeEdit" name="dateTimeEdit_intervalEnd">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Время окончания выделенного интервала</string>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>18</hour>
       <minute>0</minute>
       <second>0</second>
       <year>1999</year>
       <month>12</month>
       <day>29</day>
      </datetime>
     </property>
     <property name="displayFormat">
      <string>dd.MM.yyyy H:mm:ss</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="timeSpec">
      <enum>Qt::UTC</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_SetDtIntervalSelection">
     <property name="toolTip">
      <string>Применить изменения дат интервала</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/icons/icons/apply.png</normaloff>:/icons/icons/apply.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_CancelDtIntervalSelection">
     <property name="toolTip">
      <string>Отменить изменения дат интервала</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/icons/icons/item_error.png</normaloff>:/icons/icons/item_error.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>длительность:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_intervalLength">
     <property name="toolTip">
      <string>Длительность интервала</string>
     </property>
     <property name="decimals">
      <number>5</number>
     </property>
     <property name="maximum">
      <double>999999.999990000040270</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_IntervalTimeDim">
     <property name="toolTip">
      <string>Единицы измерения длительности интервала</string>
     </property>
     <item>
      <property name="text">
       <string>суток</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>часов</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>минут</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>секунд</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_SetIntervalLength">
     <property name="toolTip">
      <string>Применить изменения длительности интервала выделения</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/icons/icons/apply.png</normaloff>:/icons/icons/apply.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_CancelIntervalLength">
     <property name="toolTip">
      <string>Отменить изменения длительности интервала выделения</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/icons/icons/item_error.png</normaloff>:/icons/icons/item_error.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>5</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
